src/views/FAQ.vue

```diff
@@ -94,29 +91,19 @@ export default {
         ? faq_group_zero.en_heading
         : faq_group_zero[key];
     },
-    parsedText(text) {
```

Comment:
Removing this actually removes the formating of Markdown.
We generally want markdown formatting in the FAQ questions and answers.